### PR TITLE
openssl 1.0.2g in build-899

### PR DESCRIPTION
**Pull request by mischief**

### CODE REVIEW

**Comment by mischief** [comment]
fixes coreos/bugs#1160

File: profiles/coreos/base/package.accept_keywords [hunk 1]
```diff
@@ -80,11 +80,6 @@ dev-util/checkbashisms
 # https://github.com/golang/go/issues?q=milestone%3AGo1.4.3
 =dev-lang/go-1.4.3 ~amd64
 
-# 1.0.2e contains some security fixes.
-# https://bugs.gentoo.org/show_bug.cgi?id=567476
-=app-misc/c_rehash-1.7-r1 ~amd64 ~arm64
```
Comment (marineam) [inline profiles/coreos/base/package.accept_keywords]: Dropping this means you *must* update portage-stable to get the ebuild with new keywords. Considering this is a backport I'd say leave c_rehash in there.

**Comment by marineam** [comment]
Worth calling out that this is more than just the version update, it also enables ec and removes the bindist flag.

**Comment by marineam** [comment]
LGTM

**Comment by mischief** [comment]
@marineam marked c_rehash stable again.

can call out the ec change in the release notes, if it's ok to change it.

**Comment by marineam** [comment]
Yup, ok to enable ec.

**Pull request merged**